music21/romanText/writeRoman.py

```diff
@@ -332,11 +360,14 @@ def rnString(measureNumber: int,
     '''
 
     if inString:
-        inStringMeasureNumber = int(inString.split(' ')[0][1:])
-        if inStringMeasureNumber != measureNumber:
+        inStringMeasureNumber = inString.split(' ')[0][1:]
```

Comment:
> But it looks like you're concating int(measure.number) + measure.suffix only to separate them later. why not pass in the suffix separately?

Do you mean on this line? The only thing I changed was removing the cast to int. It doesn't split the suffix, it splits the inString on space and then slices the "m" from the beginning. As far as I can tell the suffix stays put once it is added.

I agree this function is clearly inefficient. It seems the obvious thing to do would be accumulate each "word" to a list and then do `' '.join` when finished.